_layouts/historical-frontend.html

```diff
@@ -0,0 +1,201 @@
+          background: #9f9
+      }
+      .diff-line-modified .diff-chunk-removed {
+          background: #f88
+      }
+
+      .diff-table td, .diff-table th { font-family: monospace; font-size: 12px; }
+      .diff-table td { border: 0; padding: 0; }
+
+      .diff-checker .diff-output .diff-line {  white-space: pre-wrap; word-break: break-all; }
+
+
+    .diff-checker .diff-table-container{display:flex;flex-direction:row;align-items:stretch}.diff-checker .diff-table-container .diff-table{clear:right;background:#fff;width:100%}.diff-checker .diff-table-container .diff-table .diff-line-inserted,.diff-checker .diff-table-container .diff-table .diff-line-modified.diff-line-with-inserts{background:#9f9;border-left:1px solid #1a981f;border-right:1px solid #1a981f}.diff-checker .diff-table-container .diff-table .diff-line-inserted.start,.diff-checker .diff-table-container .diff-table .start.diff-line-modified.diff-line-with-inserts{border-top:1px solid #1a981f}.diff-checker .diff-table-container .diff-table .diff-line-inserted.end,.diff-checker .diff-table-container .diff-table .end.diff-line-modified.diff-line-with-inserts{border-bottom:1px solid #1a981f}.diff-checker .diff-table-container .diff-table .diff-line-modified.diff-line-with-removes,.diff-checker .diff-table-container .diff-table .diff-line-removed{background:#f88;border-left:1px solid #9a2328;border-right:1px solid #9a2328}.diff-checker .diff-table-container .diff-table .diff-line-removed.start,.diff-checker .diff-table-container .diff-table .start.diff-line-modified.diff-line-with-removes{border-top:1px solid #9a2328}.diff-checker .diff-table-container .diff-table .diff-line-removed.end,.diff-checker .diff-table-container .diff-table .end.diff-line-modified.diff-line-with-removes{border-bottom:1px solid #9a2328}.diff-checker .diff-table-container .diff-table .diff-line-empty{background:#ccc!important;border-left:1px solid #888!important;border-right:1px solid #888!important}.diff-checker .diff-table-container .diff-table .diff-line-empty.start{border-top:1px solid #888!important}.diff-checker .diff-table-container .diff-table .diff-line-empty.end{border-bottom:1px solid #888!important}.diff-checker .diff-table-container .diff-table .diff-line-modified.diff-line-with-inserts{background:#e0fcd0}.diff-checker .diff-table-container .diff-table .diff-line-modified.diff-line-with-removes{background:#fcd8d9}.diff-checker .diff-table-container .diff-table .diff-line-modified .diff-chunk-inserted{background:#9f9}.diff-checker .diff-table-container .diff-table .diff-line-modified .diff-chunk-removed{background:#f88}.diff-checker .diff-table-container .diff-table td.diff-line-number{color:#666;padding:0 4px;text-align:right}.diff-checker .diff-table-container .diff-table td.diff-line-number:before{content:attr(data-content)}.diff-checker .diff-table-container .diff-table td.diff-line{margin:0 1px;width:50%}.diff-checker .diff-table-content{flex:1 1 auto}
```

Comment:
Hmm, I think there is yes. I'll see what I can do.

We copied all this CSS from a diff checker tool which outputted HTML, which is why it's not the best quality.